### PR TITLE
Feature/ashlar admin view hierarchy

### PR DESCRIPTION
**Pull request by CloudNiner**
Everything is still static, but the sidebar links all go to the proper view states.

No jQuery interactivity.

### CODE REVIEW

File: schema_editor/app/scripts/app.js [hunk 1]
```diff
@@ -24,6 +24,7 @@
      */
     angular.module('ase', [
```
Comment (kshepard) [inline schema_editor/app/scripts/app.js]: What does 'ase' stand for? I'm having trouble piecing together the acronym.

File: schema_editor/test/spec/views/sidebar/sidebar-controller.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    var rootScope;
+    var scope;
+
+    // Initialize the controller and a mock scope
+    beforeEach(inject(function ($rootScope) {
+        rootScope = $rootScope;
+        scope = $rootScope.$new();
+    }));
+
+    describe('tests for sidebar controller', function () {
```
Comment (kshepard) [inline schema_editor/test/spec/views/sidebar/sidebar-controller.spec.js]: Probably worth adding a note to the readme for how to run these tests.

**Comment by kshepard** [comment]
+1

File: schema_editor/app/scripts/app.js (hunk 1 above)
Comment (CloudNiner) [inline schema_editor/app/scripts/app.js]: Ha. Ashlar Schema Editor. Feel free to propose a new one, this one kinda sucks, but I hate having that first level be more than a few characters.

File: schema_editor/test/spec/views/sidebar/sidebar-controller.spec.js (hunk 2 above)
Comment (CloudNiner) [inline schema_editor/test/spec/views/sidebar/sidebar-controller.spec.js]: Will do.

File: schema_editor/app/scripts/app.js (hunk 1 above)
Comment (kshepard) [inline schema_editor/app/scripts/app.js]: Ah, ok. I agree, it should only be a few characters. I don't have any other great suggestions. Perhaps 'ash', but I don't feel strongly about it.

**Pull request merged**